enlighten/measurement/Measurement.py

```diff
@@ -827,6 +823,10 @@ def get_metadata(self, field):
                     log.debug(f"get_metadata: stomping {k} from plugin metadata {v}")
```

Comment:
This output message is a little confusing.  It sounds like the plugin metadata is being stomped